tests/classes/potentialSet.cpp

```diff
@@ -63,4 +64,154 @@ TEST(PotentialSetTest, ComplexAddition)
     EXPECT_EQ(2.0, pots.potentialMap()["A-D"].potential.value(0));
 }
 
+TEST(PotentialSetTest, Averaging)
+{
+    GenericList moduleData;
+    auto originalPotentialsObject =
+        moduleData.realiseIf<PotentialSet>("PotentialSet", "module", GenericItem::InRestartFileFlag);
+
+    std::string filename_{"test_restart.txt"};
```

Comment:
Rather than giving this file a generic name you could include the current test name for clarity:
```suggestion
    std::string filename_{std::format("{}.restart.txt", ::testing::UnitTest::GetInstance()->current_test_info()->name()};
```